frontend/src/components/EventForm.tsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+
+interface EventFormProps {
+  newEvent: any;
+  setNewEvent: React.Dispatch<React.SetStateAction<any>>;
+  handleFormSubmit: (e: React.FormEvent) => void;
+}
+
+const EventForm: React.FC<EventFormProps> = ({ newEvent, setNewEvent, handleFormSubmit }) => {
```

Comment:
@sanari2019 It is better to use form controls like on the other existing forms in the code. 

see for example:

```
                <Form.Group className="mb-3" controlId="formBasicEmail">
                  <Form.Label>Email address</Form.Label>
                  <Form.Control
                    className="text-muted"
                    placeholder="Email"
                    value={values.email}
                    type="email"
                    name="email"
                    data-testid="login-input-email"
                    disabled={loginRequestStatus === 'loading'}
                    onChange={handleChange}
                  />
                </Form.Group>
                
```